loader/stage0/abl/stage0_entry.S

```diff
@@ -224,9 +224,9 @@ bspstack:
 
 .balign 8
 gdt_x64:
-	.quad 	0x0000000000000000ULL	/* NULL descriptor */
```

Comment:
in the commit message, tell more about why this change is required.

Reply:
add unsupported GCC version in commit message?

Reply:
these info already in commit message.